lightning_examples/mnist-hello-world/hello-world.py

```diff
@@ -142,7 +144,7 @@ def validation_step(self, batch, batch_idx):
         logits = self(x)
         loss = F.nll_loss(logits, y)
         preds = torch.argmax(logits, dim=1)
-        acc = accuracy(preds, y)
+        acc = self.accuracy(preds, y)
```

Comment:
```suggestion
        self.accuracy(preds, y)
```

Also alter the logging below to pass in `self.acc`